plots/plots_validation.py

```diff
@@ -0,0 +1,644 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText:  Open Energy Transition gGmbH
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../")))
```

Comment:
Thanks for the update, @GbotemiB. I have placed sys and os in front to resolve the issue appeared locally. It was stating that `scripts` folder missing. This helped to resolve it.